### ISSUE TITLE
Desktop Accessibility support

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#67170]
This is the main tracking issue for the desktop accessibility support

The accessibility service does not work on any of the flutter desktop applications. The semantics updates sent to embedding are ignored at the moment, we should wire it up.

Ideally, we can fork chromium a11y layer into our code base and use it directly.

**Comment by stuartmorgan** [comment]
Is this tracking something specific different than https://github.com/flutter/flutter/issues/23601, or should it be closed as a duplicate?

**Comment by chunhtai** [comment]
I wasn't aware of that issue, i will close this one and use the old one to track the desktop a11y work
